Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -536,8 +548,8 @@ extension FirefoxHomeViewController {
 
         func cellHeight(_ traits: UITraitCollection, width: CGFloat) -> CGFloat {
             switch self {
-            case .pocket: return FirefoxHomeUX.highlightCellHeight
-            case .jumpBackIn: return FirefoxHomeUX.jumpBackInCellHeight
+            case .pocket: return FirefoxHomeUX.homeHorizontalCellHeight * 3
```

Comment:
Why are we multiplying cell height by 3 🤔 @lmarceau

Reply:
@nbhasin2 because the it's there's always three cells in a column, and I wanted to reuse the FirefoxHomeUX.homeHorizontalCellHeight. I can put the three in a variable so it's a bit clearer